resolwe/test_helpers/test_runner.py

```diff
@@ -329,6 +329,15 @@ def add_arguments(cls, parser):
             help="Do not mock purging functions (implies --keep-data)"
         )
 
+    def build_suite(self, *args, **kwargs):
+        """Build test suite."""
+        suite = super().build_suite(*args, **kwargs)
+        # Build suite first constructs the parallel suite and then may reduce self.parallel,
```

Comment:
Can you report this Django test runner's bug upstream and extend this note to explain that this is a fix for this bug (also include upstream's bug number)?

Also add a note to the CHANGELOG since this impacts Resolwe's users (e.g. Resolwe Bio developers).